src/services/review/ReviewRequestService.ts

```diff
@@ -134,7 +137,7 @@ export default class ReviewRequestService {
   }
 
   // TODO
```

Comment:
oof might be out of scope of this PR, but do you know what this todo is for?

Reply:
i don't see this specific `TODO` actually, you might have reviewed an old commit